Makefile

```diff
@@ -14,7 +14,7 @@ else
   endif
   HOST_ARCH := amd64
 endif
-undefine HOST_HARDWARE
+unexport HOST_HARDWARE
```

Comment:
That's interesting. What's the reason for this?

Reply:
I had an old version of the GNU make and it didn't work. Changed it back

Reply:
Ah, good to know. Technically, the else/undefine block isn't even necessary. So if that remains a problem we could also just remove it completely.